src/OpticalFlowEstimator.cpp

```diff
@@ -0,0 +1,598 @@
+#include "iarc7_vision/OpticalFlowEstimator.hpp"
```

Comment:
Where is the file that this file replaced?

Reply:
This didn't replace anything, Optical Flow has never been in the master branch

Reply:
So we've been running a branch of iarc7_vision on the drone the entire time?

Reply:
Yep